winterfell/middlewares/log.js

```diff
@@ -42,6 +42,22 @@ module.exports.console = function(msg) {
   console.log(msg);
 };
 
+module.exports.info = function(msg) {
```

Comment:
My personal style is towards more verbose but easier to read for people new to the code, not sure what the node convention is though